packages/terra-core-docs/package.json

```diff
@@ -51,6 +51,7 @@
     "terra-content-container": "^3.36.1",
     "terra-demographics-banner": "^3.58.2",
     "terra-dialog": "^2.63.6",
+    "terra-dialog-modal":"^3.74.5",
```

Comment:
I don't think we need terra-dialog-modal added as a dependency since it is only being referenced in the .mdx file and not actual js.